packages/node/tsconfig.json

```diff
@@ -8,10 +8,12 @@
     "lib": ["es2019"],
     "outDir": "./dist",
     "resolveJsonModule": true,
+    "skipLibCheck": true,
     "strict": true
   },
   "include": [
     "src/**/*",
+    "src/.api/**/*",
```

Comment:
i was running into build issues with the nested `.api/apis/developers/package.json` stuff so what i did instead was install `api`, `json-schema-to-ts`, etc. in `packages/node/package.json` and moved the `.api` directory into the `src/` folder so it gets properly included in the TS build

cc: @erunion

Reply:
wonder if it'd be fine if the codegen'd SDK was JS instead

Reply:
i tried both JS and TS exports and TS even more unhappy with the former